doc/tech/usecases.md

```diff
@@ -0,0 +1,194 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# CNTT Use Cases
+
+## Table of Contents
+ * [5.1 Edge](#5.1)
+   * [5.1.1 Executive Summary](#5.1.1)
+   * [5.1.2 Objective](#5.1.2)
+   * [5.1.3 Approach & Scope](#5.1.3)
+   * [5.1.4 Principle](#5.1.4)
+   * [5.1.5 Use cases](#5.1.5)
+   * [5.1.6 Terminologies](#5.1.6)
+   * [5.1.7 SDO interlocking](#5.1.7)
+   
+
+<a name="5.1"></a>
+## 5.1 Edge
+
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
+
+Edge computing, a disruptive technology, that should be considered for adoption by Digital Service Providers (DSP) as it is tightly linked with a number of use cases that monetize 5G. Thus, operators, cloud service providers, content providers and application developers are increasingly focussed on Edge computing.
+
+Edge is considered as one of the main enablers to 5G use cases adoption. There are a number of 5G challenges that Edge will support solve so as to implement the 5G requirements defined by "ITU-2020 for 5G." Edge is required to support enhanced mobile broadband (eMBB), massive machine-type communications (mMTCs) and ultra-reliable low-latency communications (uRLLCs).
```

Comment:
Please change "ITU-2020 for 5G" to "IMT-2020 and 5G" and add reference link "https://www.itu.int/pub/T-TUT-IMT"

Reply:
Done ,thx